src/module/Controller.js

```diff
@@ -1,31 +1,33 @@
-'use strict';
-
-define(function (require) {
+define(function(require) {
   class Controller {
-    list(req, res) {
+    utils = {
+      propertyUtil: require('PropertyUtil'),
+    };
+
+    list = (req, res) => {
       // Handle listing all the resources.
       res.json({ message: 'Hello from LIST' });
-    }
+    };
 
-    get(req, res) {
+    get = (req, res) => {
```

Comment:
Would it make sense if these were static methods instead?

Reply:
Opinions @alexkstrt @robinryden ?

Reply:
Hmm, wouldn't that mean that the methods can't access other things inside the object though?

Also, what is the reasoning behind using function expressions instead of normal methods? Style?

Reply:
@alexkstrt Style and no need for `.bind` in `index.js`.

Reply:
Hmm I see, what is the more standard way of writing these kind of things? Not that up-to-date with JavaScript.

Reply:
I think this is the more modern way of doing it

Reply:
In most of the use-cases in LR for example we need to access `this` to get access to things set in the parent controller. Perhaps you could set those in another manner than in the `constructor`?